packages/trader/src/Modules/SmartChart/Components/Markers/__tests__/accumulators-profit-loss-tooltip.spec.tsx

```diff
@@ -10,14 +10,15 @@ jest.mock('../accumulators-profit-loss-text', () => () => 'AccumulatorsProfitLos
 
 describe('AccumulatorsProfitLossTooltip', () => {
     const props: React.ComponentProps<typeof AccumulatorsProfitLossTooltip> = {
+        alignment: 'right',
         className: 'profit-loss-tooltip',
         currency: 'USD',
         current_spot: 6468.95,
         current_spot_time: 1666091856,
+        profit: 0.15,
```

Comment:
positive profit doesn't have a sign when it comes from API. hence, removing the sign